test fails with Go 1.24 rc2 for Ebitengine

```
panic: syscall/js: call of Value.Int on undefined

goroutine 1 [running]:
syscall/js.Value.float({{}, 0x0, 0x0}, {0xbaa3a, 0x9})
	/opt/hostedtoolcache/go/1.24.0-rc.2/x64/src/syscall/js/js.go:523 +0x10
syscall/js.Value.Int(...)
	/opt/hostedtoolcache/go/1.24.0-rc.2/x64/src/syscall/js/js.go:540
syscall.init()
	/opt/hostedtoolcache/go/1.24.0-rc.2/x64/src/syscall/fs_js.go:32 +0x37
exit with status 2
FAIL	github.com/hajimehoshi/ebiten/v2	10.742s
```

https://github.com/hajimehoshi/ebiten/actions/runs/13017255541/job/36309379923

I've used wasmbrowsertest@6c05f7170988ac62b921a5a6a4951570e0fa1b01 and Go 1.24 rc2.

I'll take a look further later.

--- a/index.html
+++ b/index.html
@@ -74,10 +74,10 @@ license that can be found in the LICENSE file.
 			return fsPath + "/" + path
 		}
 		function overrideFS(fs) {
-			// The fs.constants are read at https://github.com/golang/go/blob/561a5079057e3a660ab638e1ba957a96c4ff3fd1/src/syscall/fs_js.go#L25
-			// These values are pulled from https://github.com/golang/go/blob/561a5079057e3a660ab638e1ba957a96c4ff3fd1/src/syscall/syscall_js.go#L126
+			// The fs.constants are read at https://github.com/golang/go/blob/8071f2a1697c2a8d7e93fb1f45285f18303ddc76/src/syscall/fs_js.go#L25
+			// These values are pulled from https://github.com/golang/go/blob/8071f2a1697c2a8d7e93fb1f45285f18303ddc76/src/syscall/syscall_js.go#L126
 			fs.constants = { O_WRONLY: 1, O_RDWR: 2,
-				O_CREAT: 0o0100, O_TRUNC: 0o01000, O_APPEND: 0o02000, O_EXCL: 0o0200 };
+				O_CREAT: 0o100, O_TRUNC: 0o1000, O_APPEND: 0o2000, O_EXCL: 0o200, O_DIRECTORY: 0o20000 };
 			fs.open = (path, flags, mode, callback) => {
 				fsHandler("open", {path:fsp(path),flags,mode}, (resp) => callback(null, resp.fd), callback);
 			};

